src/message.coffee

```diff
@@ -70,7 +70,19 @@ class Message
   _onUpdateMessage: (data) ->
     @_client.logger.debug data
 
+  deleteMessage: =>
+    params = {
+      "ts": @ts,
+      "channel": @channel,
+    }
+    @_client.logger.debug "Sending message delete request"
+    @_client.logger.debug params
+    if @ts then @_client._apiCall "chat.delete", params, @_onDeleteMessage
```

Comment:
If the request is going to be conditionally sent I think the logging should go in the same conditional.